roles/vdm/tasks/assets.yaml

```diff
@@ -52,6 +68,7 @@
   with_items:
     - "{{ DEPLOY_DIR }}/sas-bases"
     - "{{ DEPLOY_DIR }}/site-config/vdm"
+    - "{{ DEPLOY_DIR }}/site-config/operator-deploy"
```

Comment:
Any reason this does not live under the vdm directory? This is some the DAC tooling handles and if we're controlling it we should place it in the Viya Deployment Management (VDM) directory. If it's documented from the Orchestration tooling to use this directory then it's fine.

Reply:
Moved this after we had the review meeting, it will now be under the `BASE_DIR/CLUSTER_NAME` directory